vscode/src/services/AuthProvider.ts

```diff
@@ -120,9 +132,18 @@ export class AuthProvider {
             return
         }
         const authState = await this.auth(instanceUrl, accessToken)
-        telemetryService.log('CodyVSCodeExtension:auth:fromToken', {
-            success: Boolean(authState?.isLoggedIn),
-        })
+        telemetryService.log(
+            'CodyVSCodeExtension:auth:fromToken',
+            {
+                success: Boolean(authState?.isLoggedIn),
+            },
+            { hasV2Event: true }
+        )
+        if (authState?.isLoggedIn) {
+            telemetryRecorder.recordEvent('cody.auth.fromToken', 'succeeded')
+        } else {
+            telemetryRecorder.recordEvent('cody.auth.fromToken', 'failed')
+        }
```

Comment:
Eliminate the redundancy here and highlight what's going on by putting the condition close to the effect it has:

```suggestion
        telemetryRecorder.recordEvent('cody.auth.fromToken', authState?.isLoggedIn ? 'succeeded' : 'failed')
```

Reply:
Implemented, thanks!